extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -38,7 +38,9 @@ const renderShoppingTagTemplate = (tagData) => (
         }
       ></span>
       <span class="amp-story-shopping-tag-pill-text">
-        {tagData['product-price']}
+        {tagData['product-tag-text']
```

Comment:
Seems like here you can use `??` or `||` depending on whether you consider the empty string to be valid `product-tag-text` or not.

Reply:
+1 to this. Good call.